templates/shared/pack.js

```diff
@@ -0,0 +1,20 @@
+const { readFileSync } = require('fs');
+const { parse, resolve } = require('path');
+const AdmZip = require('adm-zip');
+
+const { base } = parse(__dirname);
+const { version } = JSON.parse(
+  readFileSync(resolve(__dirname, 'build', 'manifest.json'), 'utf8')
+);
+
+const outdir = 'release';
+const filename = `${base}-v${version}.zip`;
+const zip = new AdmZip();
+zip.addLocalFolder('build');
+zip.writeZip(`${outdir}/${filename}`);
+
+console.log(
+  `${chalk.green(
+    'Success!'
+  )} Created a ${filename} file under ${outdir} directory. You can upload this file to web store.`
+);
```

Comment:
Let's remove the usage of `chalk.green()` from here; I thought we add it as dependency to extension but we don't. My bad.
```suggestion
console.log(
  `Success! Created a ${filename} file under ${outdir} directory. You can upload this file to web store.`
);
```